### ISSUE TITLE
Label from previous entry showing in web-app after creating alias from firefox add-on

### ISSUE THREAD

**Issue by Oryoro** [mozilla/fx-private-relay#1477]
I created an alias with the browser plugin. When I looked on the web-app, the label was showing a duplication from the preceeding alias. I was able to 'remedy' this by simply creating another blank alias in the web-app: then the name of the site I was on when I created the alias appeared in the label field instead, so I'm assuming the correct information was sent and saved in to the remote database. I could repeat the problem until I removed and re-installed the add-on. Something happened while I was creating the alias that may be relevant: The email address that was created was not populated in the email field of the website I was signing up to; it remained blank.

I have not been able to recreate the bug since. I wondered if anyone had experienced the same. Could it have been some conflict with data that is stored in the add-on compared to what is stored in the firefox-relay database? If it happens again I'll try and jot down more details.

**Comment by Vinnl** [comment]
Hi @Oryoro, is this the same issue as the one reported in #228?

**Comment by Oryoro** [comment]
@Vinnl : No that's something different. I'll try and write it more clearly. My original post is a bit messy, sorry about that.

I have a list of aliases in my account as follows:

`Alias: 0mu8bgvw@mozmail.com Label: amazon.com`
`Alias: 5403z1u5@mozmail.com Label: ebay.com`
`Alias: iq9rfauk@mozmail.com Label: netflix.com`

### Steps

1. I go to www.example.com
2. I generate an alias using the firefox add-on
3. I already have a browser tab open at 'relay.firefox.com' so I refresh it
4. The new entry should display as `Alias: dn8pbk8o@mozmail.com Label: www.example.com`
5. Instead I see `Alias: dn8pbk8o@mozmail.com Label: amazon.com` (repeats the label from the preceeding entry)

### Notes

- Refreshing the browser tab does not fix the issue
- Closing the tab, then opening a new one resolves it, so I assume the correct data is in the database.
- Creating another blank alias in the web-app also resolves it.
- While the incorrect label is being displayed: If I click in the label text, the incorrect label is saved to the database.
- When this first happened I had these browser plug-ins installed: Firefox Relay, Bitwarden, Ublock Origin, Gnome shell extensions.

**Comment by Vinnl** [comment]
@Oryoro Oh apologies, I linked just the issue number, but it's in a different repository. I meant this one: https://github.com/mozilla/fx-private-relay-add-on/issues/228

**Comment by Oryoro** [comment]
@Vinnl Yes, that sounds like the same issue. However, I'm using the paid Relay Premium, not 'Stage' (I'm assuming that's different).

**Comment by Vinnl** [comment]
Ah sorry, `stage` is where our testers can test without using actual data. Thanks for verifying!

**Comment by maxxcrawford** [comment]
Moving this issue to the other repo (and the closing as a duplicate)
